packages/terra-form-input/src/InputField.jsx

```diff
@@ -45,6 +45,10 @@ const propTypes = {
    */
   // eslint-disable-next-line react/forbid-prop-types
   inputAttrs: PropTypes.object,
+  /**
+   * Whether the field is incomplete. Field must also be required.
```

Comment:
Let's update our descriptions for isIncomplete to say:
```suggestion
   * Whether the input displays as Incomplete. Use when no value has been provided. _(usage note: `required` must also be set show display)_
```

\
Also update isInvalid to match and say:
```
   * Whether the field and input displays as Invalid. Use when value does not meet validation pattern.
```

Reply:
Add same text approach to all components (didn't add comments for all).

Reply:
Updated the `isInvalid` and `isComplete` prop descriptions for all: https://github.com/cerner/terra-core/pull/2670/commits/f3184124d28bce916fe72b76f0f19fa0aea8c0fd